dist/index.html

```diff
@@ -5,9 +5,33 @@
     <title>Fitlit</title>
   </head>
   <body>
-    <h1>Activity Tracker</h1>
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
-
+    <header>
+      <section class="title">
+        <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
+        <h1>Activity Tracker</h1>
+      </section>
+      <section class="welcome" >
+        <h2>Welcome back, Lina Quitt</h2>
+        <img src="./images/turing-logo.png" alt="turing logo" height="30px" width="30px"></imgsrc>
+      </section>
+    </header>
+    <nav>
+      <img src="./images/turing-logo.png" alt="turing logo" height="50px" width="50px"></imgsrc>
+      <img src="./images/turing-logo.png" alt="turing logo" height="50px" width="50px"></imgsrc>
+      <img src="./images/turing-logo.png" alt="turing logo" height="50px" width="50px"></imgsrc>
+      <img src="./images/turing-logo.png" alt="turing logo" height="50px" width="50px"></imgsrc>
+      <img src="./images/turing-logo.png" alt="turing logo" height="50px" width="50px"></imgsrc>
+    </nav>
+    <main>
+      <section class="top">
+        <section class="broadcast">TV</section>
+        <section class ="widgets">
+          <section class="daily-hydration-display">Hydration</section>
+          <section class="daily-sleep-display">Sleep</section>
+        </section>
+      </section>
+      <section class="bottom">Average steps in comparison to all the other users will go here</section>
```

Comment:
We will most likely need to add more elements here but we can do that when we work on this section.